### PR TITLE
Update operator description in CSV

### PR DESCRIPTION
**Pull request by amisevsk**
### What does this PR do?
Updates the operator description and begins an uninstall doc.

### Is it tested? How?

N/A

### See also
https://github.com/redhat-developer/web-terminal-operator/pull/84

### CODE REVIEW

File: doc/uninstall.md
Comment (sleshchenko) [inline doc/uninstall.md:42]: maybe it's better to do it earlier to unblock pod/exec ASAP. It can be done as soon as CRs are removed and operator is uninstalled.

**Review by sleshchenko — APPROVED** [review]
LGTM
DO you plan to fix TODOs before merging?

File: doc/uninstall.md
Comment (sleshchenko) [inline doc/uninstall.md:10]: Seems devworkspace templates are missing. It's not a requirement to remove them since they don't have finalizers yet but still what is the point to leave them on the cluster?

**Review by JPinkney — APPROVED** [review]

File: doc/uninstall.md
Comment (amisevsk) [reply doc/uninstall.md:42]: Will do. I think the reason we chose the order here is because webhooks may be re-created if the webhooks server restarts.

File: doc/uninstall.md
Comment (amisevsk) [reply doc/uninstall.md:10]: Good catch! I forgot we excluded them from WTO so they wouldn't be included in my copy-paste :)

File: doc/uninstall.md
Comment (sleshchenko) [reply doc/uninstall.md:42]: that's true. So we should remove it after webhook server is removed.

**Comment by amisevsk** [comment]
> Do you plan to fix TODOs before merging?

The remaining TODOs are for links to documentation and devfile samples. We need to fill them before release, but I don't think we have good options for this yet. Ideally linking to something like https://doc.crds.dev/github.com/devfile/api/workspace.devfile.io/DevWorkspace/v1alpha2@v2.0.0 would be great, but I don't know if we have a stable/official link to put here.

**Comment by amisevsk** [comment]
Merging this PR to get it off the backlog, opened issue https://github.com/devfile/devworkspace-operator/issues/473 to track remaining work.

**Pull request merged**
